client/packages/invoices/src/InboundShipment/api/api.ts

```diff
@@ -75,12 +76,12 @@ const inboundParsers = {
     }
   },
   toUpdate: (
-    patch: RecordPatch<InboundFragment> | RecordPatch<InboundRowFragment>
+    patch: RecordPatch<InboundFragment> | RecordPatch<InboundRowFragment> | RecordPatch<{id: string, defaultDonorId: string, updateDonorMethod: UpdateDonorMethodInput}>
```

Comment:
Wasn't sure if there was a better way to handle these type - this way felt a bit untidy. 

I wanted to not have to make a second update and point and/or a second parser.

Reply:
I think this is a great solution :) nice to be able to reuse, seeing as it calls the same endpoint